plugins/pulp_rpm/plugins/importers/yum/associate.py

```diff
@@ -102,12 +104,10 @@ def get_rpms_to_copy_by_key(rpm_search_dicts, import_conduit):
         # ignore checksum from updateinfo.xml
         key['checksum'] = None
```

Comment:
namedtuples don't have default arguments, so you have to supply a value for every parameter. So unfortunately this wouldn't work.